packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv6/ipv6.controller.js

```diff
@@ -206,4 +226,17 @@ export default class AgoraIpV6OrderController {
   goToDashboard() {
     return this.$state.go(DASHBOARD_STATE_NAME);
   }
+
+  trackClick(hit) {
+    const composit = [DEDICATED_IP_ORDER_TRACKING_PREFIX];
+
+    if (hit) {
+      composit.push(hit);
+    }
+
+    this.atInternet.trackClick({
+      name: composit.join('::'),
+      type: ACTIONS_SUFFIX,
+    });
```

Comment:
**nitpick:** Directly use string literals instead of array.
```suggestion
    this.atInternet.trackClick({
      name: `${DEDICATED_IP_ORDER_TRACKING_PREFIX}::${hit}`,
      type: ACTIONS_SUFFIX,
    });
```

Reply:
if not have a hit the track should be `DEDICATED_IP_ORDER_TRACKING_PREFIX` not `DEDICATED_IP_ORDER_TRACKING_PREFIX::` this a raison to do like that but I can duplicate call inside if statement directly and remove array